plugins/CustomVariables/config/config.php

```diff
@@ -1,9 +1,27 @@
 <?php
 
+use Piwik\Plugins\CustomVariables\Model;
+
 return array(
 
     'tracker.request.processors' => DI\add(array(
         DI\get('Piwik\Plugins\CustomVariables\Tracker\CustomVariablesRequestProcessor'),
     )),
 
+    'Piwik\Plugins\CustomVariables\Model.page' => DI\object('Piwik\Plugins\CustomVariables\Model')
+        ->constructor(Model::SCOPE_PAGE),
+    'Piwik\Plugins\CustomVariables\Model.visit' => DI\object('Piwik\Plugins\CustomVariables\Model')
+        ->constructor(Model::SCOPE_VISIT),
+    'Piwik\Plugins\CustomVariables\Model.conversion' => DI\object('Piwik\Plugins\CustomVariables\Model')
+        ->constructor(Model::SCOPE_CONVERSION),
+
+    'Piwik\Plugins\CustomVariables\Model.all' => array(
```

Comment:
Is it really needed to have it in DI like this? I know it might be nice to inject it but looks a bit over engineered / complex to me. I don't think one would want to change any thing here, does one?